toffy/batch_effects_violin_example.ipynb

```diff
@@ -0,0 +1,98 @@
+{
```

Comment:
<p>Even though only the last plot is shown here, they both get saved to the output directory right?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/153/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>